app/views/insured/group_selection/new.html.erb

```diff
@@ -101,14 +101,14 @@
         <div class="n-radio-group d-flex align-items-center my-2">
           <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'coverage_kind_health')" class="n-radio-row mr-3">
             <label class="n-radio weight-n" for="coverage_kind_health">
-              <%= radio_button_tag 'coverage_kind', 'health', is_coverage_kind_checked?("health"), disabled: is_coverage_kind_disabled?("health"), id: 'coverage_kind_health', class: 'n-radio' %>
+              <%= radio_button_tag 'coverage_kind', 'health', is_coverage_kind_checked?("health"), disabled: is_coverage_kind_disabled?("health"), id: 'coverage_kind_health', required: true, class: "n-radio" %>
               <%= l10n("health") %>
             </label>
           </div>
 
           <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'coverage_kind_dental')" id="dental-radio-button" class=" mr-3 n-radio-row <%= 'dn' if @adapter.can_shop_shop?(@person) && !@adapter.is_eligible_for_dental?(@employee_role, @change_plan, @hbx_enrollment, effective_date_for_display) %>">
             <label class="n-radio weight-n" for="coverage_kind_dental">
-              <%= radio_button_tag 'coverage_kind', 'dental', is_coverage_kind_checked?("dental"), disabled: is_coverage_kind_disabled?("dental"), id: 'coverage_kind_dental', class: 'n-radio' %>
+              <%= radio_button_tag 'coverage_kind', 'dental', is_coverage_kind_checked?("dental"), disabled: is_coverage_kind_disabled?("dental"), id: 'coverage_kind_dental', required: true, class: 'n-radio' %>
               <%= l10n("dental") %>
             </label>
           </div>
```

Comment:
Adding the required attribute to any one of the radio buttons within the group will ensure that the user must select one of them before submitting the form.

It does not cause an issue to add it all of the radio buttons, it's just not required.

No change needed here.